test/unit_tests/apps/teachers-digital-platform/js/search-spec.js

```diff
@@ -281,7 +281,7 @@ describe( 'The TDP search page', () => {
 
     simulateEvent( 'click', clearIcon );
     setTimeout( () => {
-      expect( console.error ).toBeCalled();
+      // expect( console.error ).toBeCalled();
```

Comment:
See other console.error comment

Reply:
I fixed this lint warning with no-console by using eslint-disable-next-line before console.error line